front/src/modules/ui/object/record-table/components/RecordTableHeader.tsx

```diff
@@ -67,8 +68,25 @@ const StyledTableHead = styled.thead`
 `;
 
 const StyledColumnHeadContainer = styled.div`
+  align: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
   position: relative;
   z-index: 1;
+
+  &:hover {
+    .StyledHeaderIcon {
```

Comment:
same here, could you try a useMouseEnter / useMouseLeave approach (see my comment on the other PR) to avoid this nested css. My concerns are:
-  that we will want to have a control on this hover behavior from javascript side (if the right panel is opened, we will likely want to disable this hover behavior, but that's for later)
- we avoid nested css as a code pattern so developers avoid hacking components API generally